CMakeLists.txt

```diff
@@ -251,6 +251,36 @@ else ()
   endif()
 endif ()
 
+add_executable(statistical_test
+  test/statistical_testing/graph_testing.cpp
+  test/util/graph_verifier.cpp
+  test/util/graph_gen.cpp
+  src/graph.cpp
+  src/supernode.cpp
+  src/graph_worker.cpp
+  src/work_queue.cpp
+  src/l0_sampling/sketch.cpp
+  src/l0_sampling/update.cpp
+  src/util.cpp)
+target_link_libraries(statistical_test PRIVATE xxHash::xxhash GTest::gtest)
+target_compile_definitions(statistical_test PRIVATE VERIFY_SAMPLES_F)
```

Comment:
if this is running as our cron job, shouldn't we have both with and without gutter tree?

Reply:
Done